### PR TITLE
Select contact test

### LINKED ISSUE #1520: Tests for select_contact.dart
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/after_auth_screens/chat/select_contact.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- Our `test/` folder has the same structure as the `lib` folder. Place your test file in the equivalent folder under `test/`. You may need to create the appropriate directory structure to do this.
- [Current test code coverage for this file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa?search=&displayType=list)

### IMPORTANT:
Please refer to this issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)
- No functionality must be removed to achieve this goal.

### PR DESCRIPTION
**Pull request by imshivam-gupta**
**What kind of change does this PR introduce?**

Added test for `select_contact`

**Issue Number:**

Fixes #1520 

**Did you add tests for your changes?**

Yes

**If relevant, did you update the documentation?**

No need

**Summary**

Added Test

**Does this PR introduce a breaking change?**

No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by imshivam-gupta** [comment]
@literalEval @Ayush0Chaudhary  Can you please review my PR

**Comment by imshivam-gupta** [comment]
@Cioppolo14 @palisadoes Can you please assign additional reviewers so that I can solve other issues as well?

**Review by noman2002 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

File: lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart [hunk 1]
```diff
@@ -12,20 +9,35 @@ import 'package:talawa/view_model/base_view_model.dart';
...
 class SelectContactViewModel extends BaseModel {
   late OrganizationService _organizationService;
+
+  /// orgMembersList is used to store all users of current organization.
   late List<User> orgMembersList = [];
 
-  // initialisation
+  /// This function initializes the [SelectContactViewModel] class.
+  ///
+  /// more_info_if_required
```
Comment (literalEval) [inline lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart:18]: This line should have been removed from the doc.

File: lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart [hunk 2]
```diff
@@ -12,20 +9,35 @@ import 'package:talawa/view_model/base_view_model.dart';
...
+  ///   None
+  ///
+  /// **returns**:
+  ///   None
   void initialise() {
     _organizationService = locator<OrganizationService>();
   }
 
-  /// This function is used to get all users list of an current organization.
+  /// function to get all users of current organization.
```
Comment (literalEval) [inline lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart:29]: Capitalize the first letter.

File: lib/views/after_auth_screens/chat/select_contact.dart [hunk 3]
```diff
@@ -38,15 +35,16 @@ class _SelectContactState extends State<SelectContact> {
         ),
       ),
       body: BaseView<SelectContactViewModel>(
-        onModelReady: (model) {
+        onModelReady: (model) async {
           model.initialise();
-          model.getCurrentOrgUsersList();
+          await model.getCurrentOrgUsersList();
```
Comment (literalEval) [inline lib/views/after_auth_screens/chat/select_contact.dart:40]: Why was this change necessary ? Since this was the last line of the function and no further line to wait for it, what practical change does having await here make ?

**Comment by literalEval** [comment]
@imshivam-gupta you might want to make these changes and open a new PR.

File: lib/views/after_auth_screens/chat/select_contact.dart (hunk 3 above)
Comment (imshivam-gupta) [reply lib/views/after_auth_screens/chat/select_contact.dart:40]: I was trying to debug but during testing the organizations were not being shown in the model so when I used await the test was perfectly executed.

File: lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart (hunk 1 above)
Comment (imshivam-gupta) [reply lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart:18]: Sir can you please confirm if you are talking about line no. 17,18

File: lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart (hunk 2 above)
Comment (imshivam-gupta) [reply lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart:29]: Sure Sir I will fix this doc issue as well and try to open a PR asap.

File: lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart (hunk 1 above)
Comment (literalEval) [reply lib/view_model/after_auth_view_models/chat_view_models/select_contact_view_model.dart:18]: 18
